adhocracy4/static/api.js

```diff
@@ -12,6 +12,18 @@ document.addEventListener('a4.embed.ready', init, false)
 
 const baseURL = '/api/'
 
+// This converts params = { ordering: "new" }
+// To ....endpoint/comments/?ordering=new
+const encodeQueryString = paramsObject => {
+  const params = Object.keys(paramsObject)
+  return params.reduce((acc, curr) => {
+    const symbol = acc === '' ? '?' : '&'
+    return paramsObject[curr]
+      ? `${acc}${symbol}${curr}=${paramsObject[curr]}`
+      : acc
+  }, '')
+}
```

Comment:
yo sorry for dumb question, why do we want to do this? im toatlly crap with api stuff

Reply:
ah yes, good point! this creates the querystring to sort and add filters to the api call.
Axios provides it builltin, fetch (which will be used in future) does not.

There are a couple of packages doing it, but i thought getting rid of axios and introducing something else does not help. So I have this little method to make it work. This is something we will need if we refactor this file to use fetch instead of jQuery.ajax().